### DISCUSSION TITLE
看插件帮助文档时，如何全屏观看，太小了，看的好难受

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#896] [Q&A] [answered]
![image](https://github.com/ayamir/nvimdots/assets/110797126/3fa28300-02ce-44ec-a672-2b627c27cb4f)
如上截图，，只能看下面一小部分，如何全屏展开帮助文档信息？

**Comment by Jint-lzxy** [comment, marked as answer]
`<C-w>_` or use your mouse

**Reply by wxzvim** [reply]
OK，，可以呢
